docs/details/shards/GFX/Render.md

```diff
@@ -0,0 +1,16 @@
+This function should be used only once per rendered frame.
+
+It takes a [View](../View) or sequence of [Views](../View) that represent the main camera as a parameter.
+
+??? info Default view
+    If neither `:View` nor `:Views` is specified the renderer will use a default view with both identity view and projection matrices.
+
+The `:Steps` parameter contains the list of rendering operations to perform, this can be sequence objects created by one of the following:
```

Comment:
```suggestion
The `:Steps` parameter contains the list of rendering operations to perform. This can be a sequence of objects created by one of the following:
```